.github/workflows/leancode_contracts_generator-test.yml

```diff
@@ -36,10 +36,13 @@ jobs:
         with:
           submodules: recursive
 
-      - name: Setup .NET Core SDK 6.0
-        uses: actions/setup-dotnet@v1
+      - name: Setup .NET SDK
+        uses: actions/setup-dotnet@v3
         with:
-          dotnet-version: 6.0.x
+          dotnet-version: |
+            7.0.x
```

Comment:
Fwiw .NET 7.0 is no longer supported, latest preview of backend generator requires .NET 8.0 or 9.0.